include/boost/geometry/util/range.hpp

```diff
@@ -107,25 +107,13 @@ at(RandomAccessRange & rng,
 \ingroup utility
 */
 template <typename Range>
-inline typename boost::range_value<Range>::type const&
+inline typename boost::range_reference<Range>::type
```

Comment:
To work properly also with Containers the `Range const` should be passed into the `range_reference<>`.